src/components/v1/V1Project/V1ProjectToolsDrawer/ExportTicketModsButton.tsx

```diff
@@ -0,0 +1,65 @@
+import { DownloadOutlined } from '@ant-design/icons'
+import { t, Trans } from '@lingui/macro'
+import { Button } from 'antd'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { TicketMod } from 'models/mods'
+import { useContext, useState } from 'react'
+import { downloadCsvFile } from 'utils/csv'
+import { permyriadToPercent } from 'utils/formatNumber'
+import { emitErrorNotification } from 'utils/notifications'
+
+const CSV_HEADER: (keyof TicketMod)[] = [
+  'beneficiary',
+  'percent',
+  'preferUnstaked',
+  'lockedUntil',
+]
+
+const prepareTicketModsCsv = (mods: TicketMod[]): (string | undefined)[][] => {
+  const csvContent = mods.map(mod => {
+    return [
+      mod.beneficiary,
+      `${parseFloat(permyriadToPercent(`${mod.percent}`)) / 100}`,
+      `${mod.preferUnstaked}`,
+      `${mod.lockedUntil}`,
+    ]
+  })
+
+  return [CSV_HEADER, ...csvContent]
+}
+
+export function ExportTicketModsButton() {
```

Comment:
There is some duplication here between this and ExportPayoutModsButton. I decided it wasn't worth abstracting at this stage.